themes/app/templates/Includes/HomePageAccom.ss

```diff
@@ -0,0 +1,25 @@
+<div class="accommodation container">
+    <div class="row justify-content-center">
+        <h2 class="accommodation__heading">$AccomHeading</h2>
+    </div>
+    <div class="row justify-content-center">
+        <div class="accommodation__stage col-lg-3 col-md-3 col-12" align="center">
```

Comment:
I think `col-md-3` should be sufficient here, we can remove the other two `col` classes and see if that works.